actions/actions.py

```diff
@@ -517,6 +517,19 @@ def run(self, dispatcher, tracker, domain):
             return []
 
 
+class ActionRecipients(Action):
+    def name(self):
+        return "action_recipients"
+
+    def run(self, dispatcher, tracker, domain):
+        recipients = tracker.get_slot("known_recipients")
+        formatted_recipients = "\n".join(
+            [f"- {recipient}" for recipient in recipients]
+        )
+        dispatcher.utter_message(template=utter_recipients, formatted_recipients=formatted_recipients)
```

Comment:
Oops my suggestion was missing quotes
```suggestion
        dispatcher.utter_message(template="utter_recipients", formatted_recipients=formatted_recipients)
```

Reply:
Oh also looks like this line is too long and needs to be separated out